### ISSUE TITLE
Minecraft crashes after loading 1.15.2 Nightly

### ISSUE THREAD

**Issue by Lucasino27** [ImpactDevelopment/ImpactIssues#2122]
[latest.log](https://github.com/ImpactDevelopment/ImpactIssues/files/4404122/latest.log)

**description**

When I click on play Minecraft loads and then crashes.
I have linked MC Account in dashboard so that's not the problem.

**To Reproduce**

1. Go to 'Minecraft launcher'
2. Click on 'Impact Nightly for 1.15.2'
3. Wait for it to load

**Expected behavior**
For the game to run normally with impact

**Comment by LeafHacker** [comment]
`Impact Error Code 102` implies something was invalid, but i can't see anything in the logs saying authentication failed so it could be something else. What happens running a nightly from a different night?

**Comment by ScanmanTM** [comment]
Maybe provide the crash log found in `\.minecraft\crash-reports`? 
Someone had this issue not long ago where the linked the account but they didn't have an actual minecraft account and use mcleaks which failed the auth. 
Could you provide the minecraft username to double check?

**Comment by Lucasino27** [comment]
Yeah sure, I bought minecraft like 3 days ago, nickname is : PepegusOctagus
It's not from mcleaks.
I believe it doesn't create a crash report when it crashes. I tried it 2 times to be sure but yeah, it doesn't create crash reports with this crash. (Maybe the crash reports only occur when you launch the game and it crashes during playing? idk but seems like it)
I tried to run it with nightly version 20200327 and I get the same error.
It actually worked last night, i played with it maybe for two hours, i tried baritone etc.
And then I normally turned it off and went to sleep, today morning tried to run it, but the crash keeps occuring. I can play 1.15.2 normally without crashes.
